[inductor] Use list comprehension to initialize unused_views.

ttsugriy opened the pull request:
Summary: it's more idiomatic and much more efficient

Test Plan: CI

Differential Revision: D52374626



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

Skylion007 commented:
@ttsugriy If you want to root out more of these inefficient constructs in the codebase, try enabling ruff rule PERF401

ttsugriy commented:
that's an excellent suggestion, @Skylion007 !

ttsugriy committed:
[inductor] Use list comprehension to initialize unused_views. (#116310)

Summary:
it's more idiomatic and much more efficient


Test Plan:
CI




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 aakhundov ColinPeppler

Reviewed By: htyu

Differential Revision: D52374626

Pulled By: ttsugriy
